diagnostic.md

```diff
@@ -16,13 +16,14 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/wdi/try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+A language program that allows you to communicate and edit code
```

Comment:
Not quite. GitHub is a user interface for Git. Git is a powerful application that allows us to share and publish our code, but most importantly, it provides "version control" capabilities, which helps users keep track of the changes in all parts of their project. We use Git through the command line, which employs a language called bash.